incident_viewer_demo/src/LineChart/LineChart.jsx

```diff
@@ -300,76 +297,84 @@ class LineChart extends PureComponent {
         .attr('x2', xScale(closest));
 
       const highlightedDate = moment(closest);
-      const goodYmax = yScale(incident.goodPeriodMetric);
-      const badYmax = yScale(incident.badPeriodMetric);
+      
       this.infoHoverBox.selectAll('*').remove();
 
-      if (hasIncident) {
-        const goodWidth = xScale(incident.goodPeriodEnd) - xScale(incident.goodPeriodStart);
-        const badWidth = xScale(incident.badPeriodEnd) - xScale(incident.badPeriodStart);
-        const goodHeight = plotAreaHeight-goodYmax
-        const badHeight = plotAreaHeight-badYmax
-        const incidentHeight = Math.abs(badYmax - goodYmax)
-        const rectFitsText = (goodWidth > 180) && (badWidth > 180); // NOTE: This also must be manually tuned. It hides hover text in the case 
-                                          // that the area is too small for the text to fit.
-        
-        // Draw the hover state for the good period information
-        if (highlightedDate.isBefore(incident.goodPeriodEnd) && highlightedDate.isSameOrAfter(incident.goodPeriodStart) && mouseY > goodYmax) {
-          this.infoHoverBox.append('rect')
-          .classed("good-incident-area", true)
-          .attr('x', xScale(incident.goodPeriodStart))
-          .attr('y', goodYmax)
-          .attr('width', goodWidth)
-          .attr('height', plotAreaHeight-goodYmax);
-
-          if (rectFitsText) {
-            this.infoHoverBox.append('text')
-            .classed('good-hover-text', true)
-            .attr('x', xScale(incident.goodPeriodStart) + goodWidth/2)
-            .attr('y', goodYmax + goodHeight/2)
-            .attr("alignment-baseline", "central")
-            .attr("text-anchor", "middle")
-            .text(goodDescription)
+      if (selectedASN && incidentData) {
+        for (var incIndex = 0; incIndex < incidentData[selectedASN].length; incIndex++) {
+          const goodDescription = incidentData[selectedASN][incIndex].goodPeriodInfo
```

Comment:
Add a quick comment up here that says something like "initialize incident variables"